packages/terra-framework-docs/src/terra-dev-site/doc/slide-panel/SlidePanel.1.doc.mdx

```diff
@@ -60,3 +60,13 @@ import SlidePanel from 'terra-slide-panel';
 
 ## Slide Panel Props Table 
 <SlidePanelPropsTable />
+
+## Testing
+
+The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in Jest tests teams can use `jest.mock`.
```

Comment:
```suggestion
Terra Slide Panel uses `uuid` to dynamically generate the component's description id. It can be mocked in Jest though `jest.mock`. An example is given below.
```

Reply:
Done: https://github.com/cerner/terra-framework/pull/1657/commits/c9ad6244180cd82a625d1c72c1f688c034c6c9c6